larq/optimizers_v2.py

```diff
@@ -1,10 +1,151 @@
+import warnings
+
 import tensorflow as tf
 import larq as lq
 
 from larq import utils
 from copy import deepcopy
 
 
+@utils.register_keras_custom_object
+class CaseOptimizer(tf.keras.optimizers.Optimizer):
+    """A case operation for optimizers that each train a subset of a model's variables.
```

Comment:
What do you think about:
```suggestion
    """An optmizer wrapper that applies different optimizers to a subset of variables.
```